src/main/java/org/spongepowered/api/event/entity/player/PlayerChatEvent.java

```diff
@@ -25,12 +25,21 @@
 
 package org.spongepowered.api.event.entity.player;
 
-import org.spongepowered.api.entity.player.Player;
+import org.spongepowered.api.util.command.CommandSource;
 import org.spongepowered.api.event.message.MessageEvent;
 
+import java.util.Set;
+
 /**
  * Called when a {@link Player} sends a chat message.
  */
 public interface PlayerChatEvent extends MessageEvent, PlayerEvent {
-
+    
+    /*
+     * Returns the set of {@link CommandSource}s
+     * that recieve the message. This Set is mutable.
+     *
+     * @return The {@link CommandSource}s
+     */
+    Set<CommandSource> getRecipients();
```

Comment:
Yes, it makes sense to move the method to ``MessageEvent``.
Other command sources (e.g. web chat) can possibly also send chat messages.